### ISSUE TITLE
The eSpeak DLL file is being deleted in version 2026.2 beta1

### ISSUE THREAD

**Issue by harun459** [nvaccess/nvda#20272]
### Brief summary

When installing NVDA 2026.2 beta1, Windows Defender detects eSpeak dll as a Trojan and deletes it.

### Steps to reproduce

Download 2026.2 beta1. Run the installer you downloaded and install it.

### Actual behavior

eSpeak does not work when NVDA is installed.

### Expected behavior

The eSpeak dll file should not be deleted, eSpeak should work.

### NVDA logs, crash dumps and other attachments

[nvda.log](https://github.com/user-attachments/files/28528648/nvda.log)

[nvda-old.log](https://github.com/user-attachments/files/28528651/nvda-old.log)

### NVDA type

installed copy

### NVDA version

2026.2 beta1

### Have you tried any other versions of NVDA? If so, please report their behaviors.

This problem does not occur in version 2026.1.1.

### Windows version

Windows 11 Version 25H2 (OS Build 26200.8524)

### Name and version of other software in use when reproducing the issue

None

### Other information about your system

None

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I do not have any NVDA add-ons installed

### Does the issue still occur after you run the System Accessibility Repair Tool (COM Registration Fixing Tool in older versions) in NVDA's tools menu?

I have not run the System Accessibility Repair Tool

**Comment by seanbudd** [comment]
Hi @harun459 - this can happen occasionally with releases until Defender recognises the releases. Unfortunately there's nothing we can do usually, it resolves itself before we even get a chance to contact Microsoft. We guess that once enough people install NVDA, and Defender doesn't see any dangerous red flags in practice, the pattern changes.
Does updating your Protection Updates in Windows Security settings change anything? What "Security Intelligence Version" do you have under Protection Updates?
Is this still an issue?

I can't reproduce with 1.451.240.0 for reference

**Comment by harun459** [comment]
I'm still experiencing this problem with update 1.451.240.0. I can't use Espeak because Defender deleted the Espeak DLL. Even after reinstalling the beta, Defender keeps deleting the DLL file.

**Comment by seanbudd** [comment]
we believe Microsoft has now unflagged this, can you confirm?

**Comment by harun459** [comment]
I tested it with the portable version of 2026.2 beta1. And the espeak.dll file wasn't deleted, eSpeak worked without any problems. Hopefully, Microsoft has implemented a permanent solution.
